packages/tiny-router/src/actions.ts

```diff
@@ -6,18 +6,16 @@ export const set: TinyRouter["actions"]["router"]["set"] = ({
   state,
   actions,
   libraries
-}) => routeOrParams => {
-  const { getParams, getRoute } = libraries.source;
-  const { path, page, query } = getParams(routeOrParams);
-  const route = getRoute(routeOrParams);
+}) => link => {
+  const { stringify } = libraries.source;
+  // normalizes link
+  link = stringify(link);
```

Comment:
Two things:
- Does it make sense to expose a `libraries.source.normalize` utility for these cases? I don't know if using `stringify` in a `string` makes sense. Seems confusing to me.
- Either way, if `tiny-router` is dependent on `libraries.source.stringify` or `libraries.source.normalize` then those APIs should be in `source/index.ts` instead of `wp-source/index.ts` because any source that don't implement them won't work with `tiny-router`.

Also, if finally `libraries.source.stringify` is exposed, I think `libraries.source.parse` should be exposed as well.

Reply:
Actually, now that a `link` is always a string, you would need `stringify` only when you did `parse` previously. I'm going to change `stringify` to accept only objects, and expose another utility called `nomalize` that accept strings.

I think it's a good change.

Reply:
> if tiny-router is dependent on libraries.source.stringify or libraries.source.normalize then those APIs should be in source/index.ts instead of wp-source/index.ts

They are already exposed in `source/index.ts`!

Reply:
Oh, I missed that. Sorry!